hedera-node/hedera-app/src/test/resources/old_roster.dat

```diff
@@ -0,0 +1,6 @@
+
+d
+2
+9
+gossipCaCertificate"
encryptionKey*�
+domain.com
```

Comment:
The content of this file looks bizarre, is this correct? 
```
d
2
9
...
```

Reply:
It is just a serialized data file for testing. I used the old protobuf definition which had the `tssEncryptionKey` still in it to create an instance of the Roster with the RosterEntry's and serialized it to this file. Then I read this file in the unit test and deserialized it into the an instance of the new protobuf definition to ensure it still works.  
```
 Roster.PROTOBUF.write(roster, out);
```